Protos/generate.sh

```diff
@@ -201,6 +201,26 @@ function generate_service_messages_interop_tests {
   done
 }
 
+function generate_qps_code {
```

Comment:
can we call this `generate_worker_service`?  The previous naming we used was qps, but I'm not sure where that actually came from.